devtools/platform-descriptor-json/src/main/resources/bundled-codestarts/buildtool/gradle/kotlin/build.tpl.qute.gradle

```diff
@@ -0,0 +1,32 @@
+{#include build-layout}
+{#plugins}
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version "{version.kotlin}"
+    id "org.jetbrains.kotlin.plugin.allopen" version "{version.kotlin}"
+    id '{quarkus.plugin.id}'
+}
+{/plugins}
+{/include}
+
+quarkus {
+    setOutputDirectory("$projectDir/build/classes/kotlin/main")
+}
+
+quarkusDev {
+    setSourceDir("$projectDir/src/main/kotlin")
+}
+
+allOpen {
+    annotation("javax.ws.rs.Path")
+    annotation("javax.enterprise.context.ApplicationScoped")
+    annotation("io.quarkus.test.junit.QuarkusTest")
+}
+
+compileKotlin {
+    kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8
```

Comment:
targeted java version should be a parameter imo (like pass "javaTarget" in as major version number and then each template adds what they need for it). (same for all templates below)

Reply:
Probably, as well as all the other versions in the templates. That's for the future.